docs/src/main/asciidoc/getting-started-dev-services.adoc

```diff
@@ -0,0 +1,353 @@
+
+:sectnums:
+:sectnumlevels: 3
+== Outline steps
+
+- Bootstrap the application
+- Update the application to read user input
+- Create a Panache Entity
+- Read and write the entity
+- Configure an external database using a profile
+
+== Setting up an interactive application
+
+=== Bootstrapping the project
+
+The easiest way to create a new Quarkus project is to open a terminal and run the following command:
+
+For Linux & MacOS users
+
+:create-app-artifact-id: getting-started-dev-services
+:create-app-extensions: resteasy-reactive
+:create-app-code:
+include::{includes}/devtools/create-app.adoc[]
+
+For Windows users:
+
+- If using cmd , (don't use backward slash `\` and put everything on the same line)
+- If using Powershell , wrap `-D` parameters in double quotes e.g. `"-DprojectArtifactId=getting-started"`
+
+For an explanation of what's in the generated application, see the xref:getting-started.adoc[First application guide].
+
+=== Running the application
+
+Launch the application in dev mode
+
+[source, shell]
+----
+./mvnw quarkus:dev
+----
+
+Once the application is up, visit http://localhost:8080/hello. It should show a "Hello from RestEasy Reactive" message.
+
+=== Accepting user input
+
+Let's make the application a bit more interactive.
+Open the project in your IDE and navigate to `src/main/java/org/acme/GreetingResource.java'
+Add a query param in the `hello` method.
+(The `org.jboss.resteasy.reactive.RestQuery` annotation is like the Jakarta REST `@QueryParam`
+annotation, except you don't need to duplicate the parameter name.)
+
+[source, java]
+----
+public String hello(@RestQuery String name) {
+    return "Hello " + name;
+}
+----
+
+Visit http://localhost:8080/hello?name=Bloom.
+
+You should see a personalised message: `Hello Bloom`.
+
+=== Fixing the tests
+
+In your Quarkus terminal, type 'r' to run the tests. You should see
+that your application changes broke the tests!
+
+To fix the tests, open `src/test/java/org/acme/GreetingResource.test`
+and replace
+
+[source, java]
+----
+             .body(is("Hello from RESTEasy Reactive"));
+----
+
+with
+
+[source, java]
+----
+             .body(containsString("Hello"));
+----
+
+This still validates the HTTP endpoint, but it's more flexible
+about the expected output.
+You should see in your terminal that the tests are now passing.
+
+== Adding persistence
+
+=== Creating a Panache Entity
+
+1. To add the persistence libraries, run
+
+[source, shell]
+----
+quarkus ext add hibernate-orm-panache jdbc-postgresql
+----
+
+The application will record the names of people it greets. Define a simple Hibernate Entity
+by creating a `Greeting.java` class. Add the following content:
+
+[source, java]
+----
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+import jakarta.persistence.Entity;
+
+@Entity
+public class Greeting extends PanacheEntity {
+String name;
```

Comment:
With Panache, I think we recommend to make them public:
```suggestion
    public String name;
```

Reply:
Updated, and also mirrored to the code sample in the quickstarts repo.